plugin.video.arteplussept/resources/language/German/strings.po

```diff
@@ -0,0 +1,35 @@
+# Kodi Media Center language file
```

Comment:
Why are there duplicate language files? `English` and `resource.language.en_gb `. Same goes for all languages?

Reply:
Done